src/auth/src/credentials.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 Google LLC
+// Copyright 2025 Google LLC
```

Comment:
This is not a new file, the copyright year should not change.

Reply:
ops, my bad